djangomodelimport/resultset.py

```diff
@@ -41,31 +54,37 @@ def get_counts(self):
 
 
 class ImportResultRow:
-    """ Hold the result of an imported row. """
+    """Hold the result of an imported row."""
+
     resultset = None
     linenumber = None
     row = None
     errors = None
     instance = None
     created = None
 
-    def __init__(self, resultset, linenumber, row, errors, instance, created):
+    def __init__(
+        self, resultset, linenumber, row, errors, instance, created, warnings=None
```

Comment:
Is the thinking behind this ordering to maximise backward compatibility?
Otherwise, it's a little strange to have `warnings` not next to `errors`..

Reply:
Yes, I found that the importer is used in the contractor portal and it subclasses a few things